apps/sms/views/conversation/js/conversation.js

```diff
@@ -2262,16 +2258,23 @@ var ConversationView = {
 
   // FIXME/bug 983411: phoneNumber not needed.
   simSelectedCallback: function conv_simSelected(phoneNumber, cardIndex) {
-    if (Compose.isEmpty()) {
-      return;
-    }
-
     cardIndex = +cardIndex;
```

Comment:
__nit:__ actually you need to manipulate with cardIndex only if user confirms sending, so it'd be reasonable to do like this. What do you think?

```
var sendConfirmationPromise = Compose.isEmpty() ?
  Utils.confirm(
    'sendEmptyMessage-confirmation', null, { className: 'danger' }
  ) :
  Promise.resolve();

return sendConfirmationPromise.then(() => {
  cardIndex = +cardIndex;

  if (isNaN(cardIndex)) {
    cardIndex = 0;
  }

  this.sendMessage({ serviceId: cardIndex });
});
```